README.md

```diff
@@ -511,6 +504,13 @@ Translations of the guide are available in the following languages:
 
   # good
   some_method(size, count, color)
+
+  # good - adding a parameter will not add a diff to the 'color' line
+  some_method(
```

Comment:
This conflicts with the `bad` example on line 495.

I also disagree with this style, mostly in preference:

- I think if one is breaking on parameters, it should be written with the opening and closing parens around the arguments:

```
some_method(size,
            count,
            color)
```

I think the strongest case I can make for not allowing the `newline-with-trailing-comma-arguments` is that it invites violation of open/closed. By using this style, it is expected that new arguments might be added at some point in the future to a method, which means we expect the method is not truly closed.

I do not feel especially strong about this, but it is my opinion.